website/src/components/chakra.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  ChakraProvider,
+  cookieStorageManager,
+  localStorageManager,
+} from "@chakra-ui/core"
+import { GetServerSidePropsContext } from "next"
+import theme from "theme"
+
+type WithCookies = {
+  cookies?: string
+}
+
+/**
+ * higher order component for pages
```

Comment:
I didn't include this in the examples because it's imo a special case for the docs:
when you `useColorModeValue` in the top level of a component _inline_, it will break:

```js
function MyComponent() {
  return <ChakraProvider><Box color={useColorModeValue('teal.200', 'teal.500') /></ChakraProvider>
}
```

This is done heavily in the docs, so you have to either:
- use a hoc injecting the provider around it
- refactor the docs into several subcomponents